test/fixtures/simple-app/boot/promise.reject

```diff
@@ -0,0 +1,8 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+module.exports = function(app) {
+  return Promise.reject('Promise.reject()')
```

Comment:
Actually, I think I have a better idea for how to control this. I'll do the same to test for sync exceptions also.